### PR TITLE
add hint about configurations from the extensions

### PR DESCRIPTION
**Pull request by franzholz**
Configurations are made also inside of extension files.

### CODE REVIEW

**Review by brotkrueml — COMMENTED** [review]
I am not sure that the TYPO3_CONF_VARS are meant to be set in extension configuration files as they are project-related. It might work, but is not recommended AFAIK.

**Comment by franzholz** [comment]
Many configurations are made in extension files: Setup, TSConfig TCA

**Comment by brotkrueml** [comment]
Yes, but this chapter is only about `settings.php` and `additional.php` (formerly known as `LocalConfiguration.php`  and `AdditionalConfiguraration.php`. An therefore, TYPO3_CONF_VARS. You don't put TCA or page TSconfig into one of the mentioned files.

**Comment by franzholz** [comment]
The chapter is **Configuration files**.

Therefore I would like to make a hint where to find the extension configuration files.

**Comment by brotkrueml** [comment]
Maybe, but this sentence

    These main settings supersede the configurations from the extension files

implies that you can configure TYPO3_CONF_VARS in extension files.

**Comment by franzholz** [comment]
This has always been possible to configure TYPO3_CONF_VARS in some extension PHP files.
So I should add the reference in some other way.

**Review by DavidBruchmann — CHANGES_REQUESTED** [review]
I think in this kind the link is out of context and not helping much.
I'd made an info box with content like:

    For the case that you search information about configuration of extensions you can look here [LINK]

**Comment by franzholz** [comment]
Why is this out of context, if I search for Extension configuration files and finally end on the page with the topic "configuration files"?

Why do you propose a long sentence when a "see" word would be enough to tell the user where he can also look into?

**Comment by DavidBruchmann** [comment]
Beside the "long sentence" I proposed also the info box.
Obviously you see by yourself a clear difference in topics, and grabbing attention by a box like this it would some people save perhaps some time to read a text that is unrelated to their problem or target.

**Comment by franzholz** [comment]
Maybe the sub-chapters "Global files" and "Extension files" should be introduced
as in page https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/Configuration/ConfigurationOverview.html .

**Comment by DavidBruchmann** [comment]
If you want to offer more than one link with text, let see how it's looking **in the info box**.

**Review by DavidBruchmann — APPROVED** [review]

File: Documentation/Configuration/ConfigurationFiles.rst [hunk 1]
```diff
@@ -6,6 +6,13 @@
 Configuration files
 ===================
 
+.. note::
+
+    This page is about configuration for the general TYPO3 installation.  
+    There might be many other configuration directives related to technical elements
```
Comment (brotkrueml) [inline Documentation/Configuration/ConfigurationFiles.rst:12]: What are "technical elements"? Don't know the term.

**Comment by brotkrueml** [comment]
Will provide a follow-up for this.

File: Documentation/Configuration/ConfigurationFiles.rst (hunk 1 above)
Comment (DavidBruchmann) [reply Documentation/Configuration/ConfigurationFiles.rst:12]: in the next line I wrote it, "like RTE ..."

Comment (franzholz) [reply Documentation/Configuration/ConfigurationFiles.rst:12]: "There might be many other configuration directives related to technical elements"
This sentence contains no information to me.

Comment (DavidBruchmann) [reply Documentation/Configuration/ConfigurationFiles.rst:12]: Of course, but it's not the purpose of this page to list all existing things but to inform about 2 distinct files.

Comment (brotkrueml) [reply Documentation/Configuration/ConfigurationFiles.rst:12]: Everything as a technical element, TYPO3 itself with its extensions for example.

Comment (DavidBruchmann) [reply Documentation/Configuration/ConfigurationFiles.rst:12]: Firstly the line is not the whole sentence but the next line explains it.  
Secondly I never feel that my expression is the only possible one, so feel free to replace it by another one if you think some other words could describe it better.

Comment (franzholz) [reply Documentation/Configuration/ConfigurationFiles.rst:12]: Just remove this note.
It gives no new information at all. Most users know already that it is about TYPO3 configuration. And it does not help at all to only mention other possibilities without giving links to them.

Comment (brotkrueml) [reply Documentation/Configuration/ConfigurationFiles.rst:12]: Then provide a PR for removing it again.

Comment (DavidBruchmann) [reply Documentation/Configuration/ConfigurationFiles.rst:12]: Waste of time, I mean involving many people in a PR, ignoring other people's opinions, never coming up with a useful solution and just complaining in the end.

**Pull request merged**